### PR TITLE
Add post with Trino Summit lineup

### PR DESCRIPTION
**Pull request by mosabua**
Preview at https://deploy-preview-733--trino.netlify.app/blog/2024/11/22/trino-summit-2024-lineup

### CODE REVIEW

File: _posts/2024-11-22-trino-summit-2024-lineup.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+more details. Now have a look at the whole lineup with speakers from all these
+companies:
+
+logo wall
+
+Make sure you register to get up to date information and more details for all
+the session. It will allow you to join us live, chat with the speakers during
+the event. You will also get follow up information about slide decks and
+recordings becoming available so you can review, watch anything you missed, and
+share sessions with your peers.
```
Comment (monimiller) [inline _posts/2024-11-22-trino-summit-2024-lineup.md:29]:
```suggestion
the event. You will also get important follow up information including session
recordings and slide decks so you can review, watch anything you missed, and
share sessions with your peers.
```

File: _posts/2024-11-22-trino-summit-2024-lineup.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+* Gunther Hagleitner, CEO and Co-founder at [Waii](https://waii.ai/)
+* Rong Rong, Software Engineer at [CharacterAI](https://character.ai/)
+* William Chang, Co-founder and CTO of [Canner](https://cannerdata.com/) and
+  [WrenAI](https://getwren.ai/)
+* Mustafa Sakalsiz, Founder and CEO at [Peaka](https://www.peaka.com/)
+* Dain Sundstrom, Trino co-creator and CTO at [Starburst](https://www.starburst.io/)
+
+All panelists have have extensive experience with AI and Trino, with share their
+knowledge and different perspectives.
```
Comment (monimiller) [inline _posts/2024-11-22-trino-summit-2024-lineup.md:60]:
```suggestion
All panelists have extensive experience with AI and Trino, and will share their
knowledge and different perspectives.
```

**Review by monimiller — APPROVED** [review]
LGTM after minor edits

**Comment by mosabua** [comment]
Going ahead as discussed with @martint @dain earlier today.

**Pull request merged**